### DISCUSSION TITLE
Learning Kali Linux

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69212] [Programming Help] [answered]
### Body

Hi,

I have an IQ < 70 and recently downloaded Kali as a VM and wanted to know if there is any sites to use to learn it 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebubekirgungor** [comment]
Hello, it depends on which field you are interested in. For example, if you are interested in system hacking, exploits you can learn metasploit.
Here is a metasploit tutorial video: https://www.youtube.com/watch?v=_C7nRJ6WsmI

If you want to learn using linux commands or general using of Kali linux;
here is a Kali Linux tutorial video: https://www.youtube.com/watch?v=lZAoFs75_cs

If there are fields you are interested in, I can help you if you write.

**Comment by wietyy** [comment, marked as answer]
It depends on what you want to learn. Just pick an exploitation framework, and get to it using YouTube or Google, or even chatgpt. (It might reject you, but you can persuade it to ;) )

**Comment by jddas12** [comment]
I currently installed kali linux in my laptop but wifi not showing only vpn showing there is no internet i don't have Ethernet so how can solve this problem. 
My  chipset showing realtek drive [10ec: b851]
![IMG_20250307_231805](https://github.com/user-attachments/assets/6b4b09e3-8d6a-4c41-9222-05721d613f29)
